README.md

```diff
@@ -55,69 +48,37 @@ While RFCs are intended for "significant" changes, we recommend trying to keep e
-      * A majority of non-author members have voted
-      * At least three non-author members have voted, and the vote is unanimous
-1. The RFC will be merged with the appropriate state
-
-### Reviewing an RFC
-
-Discussion should primarily take place in the pull request aimed at approving the RFC. Any discussion that takes place outside of GitHub - e.g., in meetings or community chat - should be summarized and added to the pull request.
-
-### Implementing an RFC
-
-Once an RFC has been approved, a corresponding issue should be created and prioritized accordingly in the relevant repository.
+### Writing an RFC
 
-The original author is welcome but not obligated to implement it. If you are interested in implementing an RFC, please mention it in either the RFC pull request or the new issue (depending on which is active by then). While authors who are interested in implementation generally get priority, we encourage anyone who's interested to collaborate together!
+* First, [fork](https://help.github.com/en/articles/fork-a-repo) this [repo](https://github.com/open-telemetry/oteps).
+* Copy [`0000-template.md`](./0000-template.md) to `text/0000-my-rfc.md`, where `my-rfc` is a title relevant to your proposal, and `0000` is the RFC ID. Leave the number as is for now. Once a Pull Request is made, update this ID to match the PR ID.
+* Fill in the template. Put care into the details: It is important to present convincing motivation, demonstrate an understanding of the design's impact, and honestly assess the drawbacks and potential alternatives.
 
-If neither an author nor another community member asks to implement the RFC, it will be assigned once it gets high enough in the backlog.
+### Submitting the RFC
+* An RFC is `proposed` by posting it as a PR. Once the PR is created, update the RFC file name to use the PR ID as the RFC ID.
+* An RFC is `approved` when four reviewers github-approve the PR. The RFC is then merged.
+* If an RFC is `rejected` or `withdrawn`, the PR is closed. Note that these RFCs submissions are still recorded, as Github retains both the discussion and the proposal, even if the branch is later deleted.
+* If an RFC discussion becomes long, and the RFC then goes through a major revision, the next version of the RFC can be posted as a new PR, which references the old PR. The old PR is then closed. This makes RFC review easier to follow and participate in.
 
-### RFCs vs GitHub issues
+### Integrating the RFC into the Spec
+* Once an RFC is `approved`, an issue is created in the [specification repo](https://github.com/open-telemetry/opentelemetry-specification) to integrate the RFC into the spec.
+* When reviewing the spec PR for the RFC, focus on whether the spec is written clearly, and reflects the changes approved in the RFC. Please abstain from relitigating the approved RFC changes at this stage.
+* An RFC is `integrated` when four reviewers github-approve the spec PR. The PR is then merged, and the spec is versioned.
```

Comment:
Do we enforce one single spec PR for each RFC, or we allow (or even encourage) multiple smaller PRs?

Reply:
Hmmmm, that's a good question. Hopefully RFCs are tightly scoped enough that it would not be an issue to make a single PR. If multiple PRs were needed, I think that would be fine... so long as we don't accidentally release a version of the spec with half an RFC!!